src/realm-settings/PoliciesTab.tsx

```diff
@@ -58,10 +59,62 @@ export const PoliciesTab = () => {
 
   const loader = async () => policies ?? [];
 
+  const saveStatus = async (updatedPolicy: ClientPolicyRepresentation) => {
+    const switchValues = form.getValues();
+
+    const updatedPolicies = policies?.map((policy) => {
+      if (policy.name !== updatedPolicy.name) {
+        return policy;
+      }
+
+      const enabled = switchValues[updatedPolicy.name!];
+
+      return {
+        ...policy,
+        enabled: enabled,
+      };
+    }) as ClientPolicyRepresentation[];
```

Comment:
Instead of using `as` (which might cast the type invalidly) you can provide a generic of the expected type to `map()`.
```suggestion
    const updatedPolicies = policies?.map<ClientPolicyRepresentation>(
      (policy) => {
        if (policy.name !== updatedPolicy.name) {
          return policy;
        }

        const enabled = switchValues[updatedPolicy.name!];

        return {
          ...policy,
          enabled: enabled,
        };
      }
    );
```